### ISSUE TITLE
cp: no such file or directory: scripts/validate-commit-msg.js

### ISSUE THREAD

**Issue by bugwelle** [ajoslin/angular-scrolly#11]
After forking and running `npm install` and `bower install`, I tried to build the project. I get this error:

``` bash
$ grunt build
Running "build" task
cp: no such file or directory: scripts/validate-commit-msg.js

Warning: ENOENT, no such file or directory '.git/hooks/commit-msg' Use --force to continue.
```

Could you check the lines 153-156 in `Gruntfile.js`, please?

Thanks,
Andre

**Comment by ajoslin** [comment]
Oh, forgot to `git add` it. Fixed in 75816610e0d2496ca64a64c40582f069d4bd9224

**Comment by bugwelle** [comment]
:+1: Thanks.
